wdae/wdae/remote/remote_phenotype_data.py

```diff
@@ -241,8 +246,12 @@ def get_measures_info(self):
         output = self.rest_client.get_browser_measures_info(
             self.remote_dataset_id
         )
-        output["base_image_url"] = \
-            f"/{self.rest_client.gpf_prefix}/{output['base_image_url']}"
+        remote_base = output["base_image_url"][:-1]
+        pheno_folder = remote_base[remote_base.rindex("/") + 1:]
+        output["base_image_url"] = (
+            "/api/v3/pheno_browser/remote_images/"
+            f"{self.rest_client.remote_id}/{pheno_folder}/"
+        )
```

Comment:
The parsing of `base_image_url` seems frigile. 

What happens when `base_image_url` ends with `//` or if it has no `/` at the end.

Could you please create a function that does this transformation and add tests for it.